### PR TITLE
Bug 1219161 - Make sure the statusbar iconContainer is available during FTU r=albertopq

### PR DESCRIPTION
**Pull request by etiennesegonzac**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2c6c1e4b3ed6fe1f48b313f102ae4f20ebca128e

File: apps/system/js/statusbar.js [hunk 1]
```diff
@@ -92,6 +92,7 @@
         window.addEventListener('ftudone', this);
       }
       Service.registerState('height', this);
+      Service.register('iconContainer', this);
```
Comment (etiennesegonzac) [inline apps/system/js/statusbar.js]: we can expose this as soon as `getAllElements` was called

File: apps/system/js/statusbar.js [hunk 2]
```diff
@@ -160,8 +161,6 @@
         console.error('UtilityTray load or init error', err);
       });
 
-      Service.register('onIconCreated', this);
```
Comment (etiennesegonzac) [inline apps/system/js/statusbar.js]: we don't have an implementation for this one, nor a user :)

Comment (albertopq) [inline apps/system/js/statusbar.js]: Sorry, forgot to remove it

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8d9ec69c9940dcfb4b6f51aceef37d89927335c3

**Pull request merged**